src/commands/help.ts

```diff
@@ -46,6 +46,10 @@ module.exports = {
       ["ignite doctor"],
     )
     p()
+    command("rename      ", "Renames your React Native project (experimental)", [
+      "ignite rename NewName com.mycompany.newname",
+    ])
```

Comment:
I know its experimental, but would be good to add visibility to the README file of this command

Reply:
I'm working on another branch with a bunch of new documentation and will definitely add it there. It doesn't really have a place to live right now, unfortunately. Good catch though!